src/screens/faq/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { useTranslation } from 'i18n';
+import {
+  Layout, MaxWidthContainer, MaxWidthContent,
+} from '@components';
+import Grid from '@material-ui/core/Grid';
+import {
+  createMuiTheme, MuiThemeProvider,
+} from '@material-ui/core/styles';
+import { useGetStyles } from './styles';
+
+const themeLocal = createMuiTheme({
+  breakpoints: {
+    keys: ['xs', 'sm', 'md', 'lg', 'xl'],
+    values: {
+      xs: 0, sm: 568, md: 760, lg: 1080, xl: 1200,
+    },
+  },
+});
```

Comment:
I know what you are trying to do but it's not advised to create a theme within a theme! We already have a global one that you can edit in `src/styles/theme.ts` I also think it would be better if we create a custom grid component with out own grid breakpoints as it will be used across the app.